crypto/key_store_passphrase.go

```diff
@@ -112,40 +113,44 @@ func (ks keyStorePassphrase) GetKey(keyAddr []byte, auth string) (key *Key, err
 	return key, err
 }
 
-func (ks keyStorePassphrase) GetKeyAddresses() (addresses [][]byte, err error) {
+func (ks keyStorePassphrase) GetKeyAddresses() (addresses []common.Address, err error) {
 	return GetKeyAddresses(ks.keysDirPath)
 }
 
 func (ks keyStorePassphrase) StoreKey(key *Key, auth string) (err error) {
 	authArray := []byte(auth)
-	salt := randentropy.GetEntropyMixed(32)
+	salt := randentropy.GetEntropyCSPRNG(32)
```

Comment:
It panics if no random data can be read: https://github.com/ethereum/go-ethereum/blob/47feff361113fd4b28f81f7b7094c4662b4e8786/crypto/randentropy/rand_entropy.go#L70